new-frontend/src/app/pages/task/create/task-create.component.ts

```diff
@@ -209,6 +260,18 @@ export class TaskCreateComponent implements OnInit, OnDestroy {
     }
   }
 
+  private async getOnlineNode(): Promise<BaseNode | null> {
+    //Get all nodes for chosen collaboration
+    await this.getNodes();
+
+    //Find a random node that is online
+    return this.collab_nodes.find((_) => _.status === 'online') || null;
```

Comment:
State variable can be removed, directly return the results as in the cases where the data is used it is only directly used.

Reply:
Collab_nodes variable has been removed